src/main/java/Main.java

```diff
@@ -0,0 +1,78 @@
+import Controller.GameController;
+import Controller.InitController;
+import entity.*;
+import usecase.*;
+
+
+public class Main {
+    public static GameController controller;
+    public static GameData data;
+
+
+//    public static final String propertiesFile = "data/properties.csv";
+    public static final String propertiesFile = "data/test/properties_example.csv";
+
```

Comment:
I think we should move the data files into a new folder "develop", and create some new small-size data for the test

Reply:
I think that makes sence, we should use multiple datas to test the game thoroughly

Reply:
Alwyn has moved test data into data/test folder. You mean we need more test folders?

Reply:
I think the files are used for initialization now. We need some small-size data to test the generator